tests/v1/kv_connector/nixl_integration/run_accuracy_test.sh

```diff
@@ -85,12 +85,15 @@ run_tests_for_model() {
     echo "Starting prefill instance $i on GPU $GPU_ID, port $PORT"
 
     # Build the command with or without model-specific args
```

Comment:
can we revert changes to the testing script? roles can be confusing as per-above

Reply:
ok, will do . although explictly claim the role will make things clear , but just keep `kv_both` is also fine.